### PR TITLE
Added modular fellows program

### PR DESCRIPTION
**Pull request by dabit3**

### CODE REVIEW

File: docs/community/modular-fellows.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# Modular Fellows
+
```
Comment (YazzyYaz) [inline docs/community/modular-fellows.md:2]: Perhaps we can add a banner image here?

File: docs/community/modular-fellows.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+- **Telegram group** - this group will be for open discussion during the current
+  cohort
+- **Modular fellow role on Discord** - you will be granted this role upon acceptance
+  into the program
+- **Slack channel** - upon acceptance to the program you will be invited to join
+  this channel
+- **Twitter** - We will share updates about the program and participants via our
+  Twitter (tbd - and Developer Twitter accounts)
+- **Discord** - Members of the Modular Fellows program will also be given a
```
Comment (YazzyYaz) [inline docs/community/modular-fellows.md:28]: Isn't this repeating the second point?

File: docs/community/modular-fellows.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  special role in Discord.
+
+### Milestones
+
+- **Month 1** - learning and exploring (start building whenever you’re ready)
```
Comment (YazzyYaz) [inline docs/community/modular-fellows.md:33]: I believe this should be a title "Learning and Exploring"

File: docs/community/modular-fellows.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  - In this month, you will immerse yourself in modular blockchains to come up
+    with ideas of protocols, tools or applications to build during the rest
+    of the program.
+  - *Milestone - present an idea that you want to pursue*
+- **Month 2** - designing & implementing a basic prototype
```
Comment (YazzyYaz) [inline docs/community/modular-fellows.md:38]: "Designing and Implementing a Basic Prototype"

File: docs/community/modular-fellows.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+  - In this month, you will design and implement a basic prototype of the
+    idea that you settled on during the first month.
+  - *Milestone - present the prototype for feedback*
+- **Month 3** - build out the prototype into a functional product
```
Comment (YazzyYaz) [inline docs/community/modular-fellows.md:42]: "Build Out the Prototype into a Functional Product"

File: docs/community/modular-fellows.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+Fellows will receive $5,000.00 per month paid on completion of each milestone.
+
+### Contacts
+
+For any questions, reach out to Nader Dabit via email nader@celestia.org or
+Telegram @naderdabit.
+
+### Continuation
+
+- **Grants & Funding -** If you’d like to continue to build your project after the
```
Comment (YazzyYaz) [inline docs/community/modular-fellows.md:77]: Move the dash to be after the **

File: docs/community/modular-fellows.md (hunk 1 above)
Comment (dabit3) [reply docs/community/modular-fellows.md:2]: Working on this now with Alex

**Review by YazzyYaz — APPROVED** [review]

**Pull request merged**